resources/Notices.php

```diff
@@ -30,18 +39,30 @@ public function addHooks()
     }
 
     /**
-     * Check if gravityforms is active. If not, issue a notice
+     * Check if gravityforms is installled and active. If not, issue an activation or installation notice
      */
     public function isGravityFormsActive($inline = '', $alt = '')
     {
-        if (!class_exists('GFAPI')) {
-            $notice = sprintf(__('Warning: You need to <a href="%s">Activate Gravityforms</a> in order to use the Advanced Custom Fields: Gravityforms Add-on.',
-                ACF_GF_FIELD_TEXTDOMAIN), admin_url('plugins.php'));
+        $notice = "";
 
-            $this->createNotice($notice, $inline, $alt);
+        if ($this->utils->isPluginInstalled("Gravity Forms")) {
+            if (!class_exists('GFAPI')) {
+                $activateUrl = $this->utils->generatePluginActivationLinkUrl('gravityforms/gravityforms.php');
+
+                $notice = sprintf(__('Warning: You need to <a href="%s">Activate Gravityforms</a> in order to use the Advanced Custom Fields: Gravityforms Add-on.',
+                    ACF_GF_FIELD_TEXTDOMAIN), $activateUrl);
+            }
+        } else {
```

Comment:
The method isGravityFormsActive uses an else expression. Else is never necessary and you can simplify the code to work without else.  …